CTX.js

```diff
@@ -271,14 +213,30 @@ var mapXMLtoKEV = {
 	'info:ofi/fmt:xml:xsd:patent'	:	'info:ofi/fmt:kev:mtx:patent',		// Patents
 	'info:ofi/fmt:xml:xsd:sch_svc'	:	'info:ofi/fmt:kev:mtx:sch_svc'		// Scholarly ServiceTypes
 };
+
+function getItems() {
+	return items;
+}
+
+function getZoteroItems() {
+	return zoteroItems;
+}
+
+var exports = {
+	"doImportFromText": doImportFromText,
+	"getItems" : getItems,
+	"getZoteroItems" : getZoteroItems
+}
```

Comment:
We shouldn't need to do this, unless I'm overlooking something. Any translator that wants to import CTX should set the XML string as input to the CTX translator and wait for `itemDone` handler to get called with parsed items.